alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py

```diff
@@ -6,22 +6,22 @@
 
 """
 
+from alembic import op
+
 # revision identifiers, used by Alembic.
 revision = '558ac277c2b9'
 down_revision = '341ee10697f1'
 
-from alembic import op
-import sqlalchemy as sa
-
 
 def upgrade():
     pass
-    #op.create_table('bundle_action',
```

Comment:
Can we just get rid of commented out code?

Reply:
sure, I can do it.

Reply:
done